src/main/java/Task.java

```diff
@@ -0,0 +1,35 @@
+public class Task {
+    protected String task;
+    protected boolean done;
```

Comment:
```suggestion
    protected boolean isDone;
```
This boolean variable should be named to sound like booleans.